modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm

```diff
@@ -522,16 +522,16 @@
 	return TRUE
 
 /datum/antagonist/bloodsucker/proc/forge_bloodsucker_objectives()
-	// Claim a Lair Objective
-	var/datum/objective/bloodsucker/lair/lair_objective = new
-	lair_objective.owner = owner
-	objectives += lair_objective
+	// Claim a haven Objective
+	var/datum/objective/bloodsucker/haven/haven_objective = new
+	haven_objective.owner = owner
+	objectives += haven_objective
 	// Survive Objective
 	var/datum/objective/survive/bloodsucker/survive_objective = new
 	survive_objective.owner = owner
 	objectives += survive_objective
 
-	// Objective 1: Vassalize a Head/Command, or a specific target
+	// Objective 1: Ghoulize a Head/Command, or a specific target
```

Comment:
Ghoul, not ghoulize

Reply:
Ghoulification sounds better. I'll go w/ that.